lib/matplotlib/scale.py

```diff
@@ -79,6 +79,18 @@ def limit_range_for_scale(self, vmin, vmax, minpos):
         return vmin, vmax
 
 
+class _TransformScaleCopyable():
```

Comment:
```suggestion
class _CopyableTransfromMixin():
    """Mixin to support deep copy on a transfrom."""
```

I don't think we need "scale" in here. It's just the copy feature that we provide and doesn't have to say anything on who may use it.

While historically, we did not call mixins `*Mixin`, I think it's good style to do so, and there is precedence with `_BlendedMixin`.

Reply:
sure, plus or minus spelling ;-)